cob/bootstrapping.py

```diff
@@ -70,7 +69,7 @@ def _ensure_virtualenv():
         if os.environ.get(_USE_PRE_ENV_VAR):
             args.append('--pre')
         if PYPI_INDEX_ENV_VAR in os.environ:
-            args.extend(['-i'], os.environ[PYPI_INDEX_ENV_VAR])
```

Comment:
WHAT? did that even run without throwing an exception???

Reply:
Yep